doc/classes/String.xml

```diff
@@ -288,43 +328,44 @@
 			<return type="String" />
 			<param index="0" name="size" type="int" />
 			<description>
-				Converts an integer representing a number of bytes into a human-readable form.
-				Note that this output is in [url=https://en.wikipedia.org/wiki/Binary_prefix#IEC_prefixes]IEC prefix format[/url], and includes [code]B[/code], [code]KiB[/code], [code]MiB[/code], [code]GiB[/code], [code]TiB[/code], [code]PiB[/code], and [code]EiB[/code].
+				Converts [param size] which represents a number of bytes into a human-readable form.
+				The result is in [url=https://en.wikipedia.org/wiki/Binary_prefix#IEC_prefixes]IEC prefix format[/url], which may end in either [code]"B"[/code], [code]"KiB"[/code], [code]"MiB"[/code], [code]"GiB"[/code], [code]"TiB"[/code], [code]"PiB"[/code], or [code]"EiB"[/code].
 			</description>
 		</method>
 		<method name="indent" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="prefix" type="String" />
 			<description>
-				Returns a copy of the string with lines indented with [param prefix].
-				For example, the string can be indented with two tabs using [code]"\t\t"[/code], or four spaces using [code]"    "[/code]. The prefix can be any string so it can also be used to comment out strings with e.g. [code]"# "[/code]. See also [method dedent] to remove indentation.
-				[b]Note:[/b] Empty lines are kept empty.
+				Returns a copy of the string with lines indented with [param prefix]. Empty lines are kept empty. See also [method dedent] to remove indentation.
+				For example, the string can be indented with two tabs using [code]"\t\t"[/code], or four spaces using [code]"    "[/code]. The [param prefix] can be any [String], so it can also be used to comment out strings with e.g. [code]"# "[/code].
 			</description>
 		</method>
 		<method name="insert" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="position" type="int" />
 			<param index="1" name="what" type="String" />
 			<description>
-				Returns a copy of the string with the substring [param what] inserted at the given [param position].
-			</description>
+				?? Returns a copy of the string with the [param what] inserted at the given [param position].
+				?? Inserts [param what] inside the string, at the given [param position].
```

Comment:
The last doubt is this one. I am not sure which exactly is best.

Reply:
I prefer the 2nd one, though I think neither flow perfectly. How about "Inserts [what] at the given [position] in the string"?

~~this is the 69th comment on this thread yipee~~